File: OCH_Tao/ABC/B/121.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [N, M, C] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
const B = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
let cnt = 0;
for (const X of input) {
  const A = (X.split(" ")).map(x => parseInt(x, 10));
  let tmp = 0;
  for (let i = 0; i < M; i++) {
    tmp += A[i] * B[i];
  }
  tmp += C;
  if (tmp > 0) {
    cnt++;
  }
}
console.log(cnt);

File: OCH_Tao/ABC/B/121.py
N,M,C = map(int,input().split())
B = list(map(int,input().split()))
cnt = 0
for i in range(N):
  A = list(map(int,input().split()))
  tmp = 0
  for j in range(M):
    tmp+=A[j]*B[j]
  else:
    tmp+=C
    if tmp>0:
      cnt+=1
print(cnt)